src/apis/serverActions.js

```diff
@@ -0,0 +1,30 @@
+const BASE_URL = "https://boolean-api-server.fly.dev/rafa-lopes-pt/contact";
+
+export const getAllEntries = async () => fetch(BASE_URL);
+
+export const addNewEntry = async (data) =>
+	fetch(BASE_URL, {
+		method: "POST",
+		body: JSON.stringify(data),
+		headers: {
+			"Content-Type": "application/json",
+		},
+	});
+
+export const updateEntry = async (data) =>
+	fetch(BASE_URL + "/" + data?.id, {
+		method: "PUT",
+		body: JSON.stringify(data),
+		headers: {
+			"Content-Type": "application/json",
+		},
+	});
+
+export const delEntry = async (data) => {
+	fetch(BASE_URL + "/" + data?.id, {
+		method: "DELETE",
+		headers: {
+			"Content-Type": "application/json",
+		},
+	});
+};
```

Comment:
I would take the abstraction a little further here and make a function that forms the fetch:

```js
const sendRequest = (path, method, data = null) => {
    const options = {
		method,
		headers: {
			"Content-Type": "application/json",
		},
    }

    if (data !== null) options.body = JSON.stringify(data)

    return fetch(BASE_URL + "/" + path, options);
}
```

Then you have a simple way to create your specific functions for individual calls